phpstan.neon

```diff
@@ -25,5 +25,12 @@ parameters:
         - %currentWorkingDirectory%/*/vendor/*
         - %currentWorkingDirectory%/bin/console
         - %currentWorkingDirectory%/src/Sulu/Bundle/TestBundle/Resources/app/config/config.php
+        # the following files are excluded because of autoloader error which cant be add to ignoreErrors
```

Comment:
Is there really no better approach? :thinking:

Reply:
Lets me check if the errors still exist. At the state when developed I discussed it with phpstan developer and the only other solution would be create the not exist third party classes as stub classes in our repository.

Reply:
Could remove some files here but the RestController need to stay. Creating a stub I find a real overhead also then it need to be handled that the stub is only loaded when using Symfony 5 and not Symfony 4.4. Also the RestController is deprecated since 2.0 and can not be used with Symfony 5, its just there for projects which maybe did use it.